napari/_qt/tests/test_qt_dims.py

```diff
@@ -1,140 +1,138 @@
-import os
 import numpy as np
-from napari.components import Dims
-from napari._qt.qt_dims import QtDims
-from napari import gui_qt
 
-os.environ['NAPARI_TEST'] = '1'
+from ...components import Dims
+from ..qt_dims import QtDims
```

Comment:
Typically we use absolute imports in tests. This allows testing of installed packages e.g. if we ever decide not to ship the testing code with the application.